test_scripts/API/VehicleData/StabilityControlsStatus/002_SubscribeVehicleData_StabilityControlsStatus.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check subscription on StabilityControlsStatus data via SubscribeVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+
+-- Steps:
+-- 1) App send SubscribeVehicleData (with stabilityControlsStatus = true) request to SDL
+-- SDL does:
+--  - send VehicleInfo.SubscribeVehicleData (with stabilityControlsStatus = true) request to HMI
+-- HMI sends VehicleInfo.SubscribeVehicleData response "SUCCESS"
+--   with next data (stabilityControlsStatus = { dataType = "VEHICLEDATA_STABILITYCONTROLSSTATUS"} )
+-- SDL does:
+--  - send SubscribeVehicleData response with (success: true resultCode: "SUCCESS") and received from HMI data to App
+-- 2) HMI sends VehicleInfo.OnVehicleData notification with StabilityControlsStatus data
+--   (escSystem = "ON", trailerSwayControl = "OFF")
+-- SDL does:
+--  - send OnVehicleData notification with received from HMI data to App
+-- 3) HMI sends VehicleInfo.OnVehicleData notification with GPS data
+-- SDL does:
+--  - not send OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+-- local common = require('test_scripts/API/VehicleData/StabilityControlsStatus/commonVDStabilityControlsStatus')
```

Comment:
Please remove redundant require here and in the other tests below

Reply:
@VjKlepikov Fixed (see a7a20fe)